cocos/2d/CCSpriteFrameCache.cpp

```diff
@@ -79,6 +80,63 @@ SpriteFrameCache::~SpriteFrameCache()
     CC_SAFE_DELETE(_loadedFileNames);
 }
 
+void SpriteFrameCache::parseIntegerList(const std::string string, std::vector<int> &res)
+{
+    std::string delim(" ");
+
+    size_t n = std::count(string.begin(), string.end(), ' ');
+    res.resize(n+1);
+    
+    size_t start  = 0U;
+    size_t end = string.find(delim);
+    
+    int i=0;
+    while (end != std::string::npos)
+    {
+        res[i++] = atoi(string.substr(start, end - start).c_str());
+        start = end + delim.length();
+        end = string.find(delim, start);
+    }
+    
+    res[i] = atoi(string.substr(start, end).c_str());
+}
+
+void SpriteFrameCache::initializePolygonInfo(const Size &textureSize,
+                                             const Size &spriteSize,
+                                             const std::vector<int> &vertices,
+                                             const std::vector<int> &verticesUV,
+                                             const std::vector<int> &triangleIndices,
+                                             PolygonInfo &info)
+{
+    size_t vertexCount = vertices.size();
+    size_t indexCount = triangleIndices.size();
+    
+    float scaleFactor = CC_CONTENT_SCALE_FACTOR();
+
+    V3F_C4B_T2F *vertexData = new V3F_C4B_T2F[vertexCount];
```

Comment:
when does the structure memory of `vertexData` got released? I think it's the polygonInfo's responsibility to do the house clean work. So we should set `_isOwner` to true inside polygon info, thoughts?